demos/pet-demo/src/components/PetCheckinsCard.tsx

```diff
@@ -50,7 +50,7 @@ export const CheckinDisplay = iso(`
 });
 
 export const PetCheckinsCardList = iso(`
-  field Pet.PetCheckinsCardList($skip: Int, $limit: Int) {
+  field Pet.PetCheckinsCardList($skip: Int!, $limit: Int!) {
```

Comment:
useSkipLimitPagination hook expects it to be required

Reply:
love that this is catching bugs!